libafl/src/inputs/multi.rs

```diff
@@ -154,16 +154,16 @@ where
     I: Input,
 {
     fn generate_name(&self, id: Option<CorpusId>) -> String {
```

Comment:
Not related to this PR, but shouldn't/couldn't `generate_name` return a `Cow<'static str>` as well?

Reply:
cc @R9295